test/controllers/articles_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class ArticlesControllerTest < ActionController::TestCase
+  setup do
+    @article = articles(:one)
+  end
+
+  test "should get index" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.